### ISSUE TITLE
Ods writer does not save dimentions (width/height)

### ISSUE THREAD

**Issue by vovka1200** [PHPOffice/PhpSpreadsheet#1418]
This is:

```
- [X] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?

changed columns/rows dimensions in ODS

### What is the current behavior?

autoSize and setWidth/SetROwHeight does work in PhpSpreadsheet, but Ods writer ignores them

### What are the steps to reproduce?

```php
<?php
require 'vendor/autoload.php';

use PhpOffice\PhpSpreadsheet\IOFactory,
    PhpOffice\PhpSpreadsheet\Spreadsheet;

$spreadsheet = new Spreadsheet();
$sheet       = $spreadsheet->getActiveSheet();
$sheet->getCell("A1")->setValue("test");
$sheet->getColumnDimension('A')->setWidth(20.0);
$sheet->getRowDimension(1)->setRowHeight(50.0);

IOFactory::createWriter($spreadsheet, "Ods")->save("test.ods");  // width, heigh are default
IOFactory::createWriter($spreadsheet, "Xlsx")->save("test.xlsx"); // width, height are as expected
```

### Which versions of PhpSpreadsheet and PHP are affected?

composer "phpoffice/phpspreadsheet": "dev-master" , 1.11.0
PHP 7.2.24

**Comment by alcalyn** [comment]
I'm experiencing the same bug

**Comment by ppc52776** [comment]
I have the same bug.

**Comment by alcalyn** [comment]
The thing I finally did is to use only `.xlsx` export and open it with LibreOffice, gives better results

**Comment by ppc52776** [comment]
My solution is using libreoffice to convert `.xlsx` to `.ods`. The ods writer is missing a lot of format support.
The command used for converting is 
`soffice --headless --convert-to ods example.xlsx`
The output will be `example.ods`

**Comment by alcalyn** [comment]
Ah, nice trick !

**Comment by italotasso** [comment]
This bug is still present in 1.17.1.

**Comment by tobias47n9e** [comment]
Just ran into this issue. Issue seems to need reopening.

**Comment by Kest007** [comment]
This bug is still present in 1.21.0. =(

**Comment by oleibman** [comment]
Fixed by PR #2753 in April 2022. Removing stale label, leaving issue closed.
